docs/content/standards/kiosk-apps/nft-rental.mdx

```diff
@@ -0,0 +1,428 @@
+
+### Ticketing
+
+In the realm of ticketing, NFTs play a pivotal role in enhancing transferability. These digital assets facilitate a secure and traceable transfer, resale, or rental of tickets, mitigating the risk of counterfeit tickets within the secondary market. The blockchain-based nature of NFTs ensures transparency and authenticity in each transaction, providing users with a reliable and fraud-resistant means to engage in ticket-related activities. This innovation not only simplifies the process for ticket holders but also contributes to a more trustworthy and efficient secondary ticket market.
+
+### Virtual land {#virtual-land}
+
+Renting virtual lands and offices in the metaverse provides businesses with flexible solutions, enabling event companies to host gatherings without the commitment of permanent acquisitions and facilitating remote work through virtual offices. This approach not only offers cost-effective alternatives but also aligns with the evolving dynamics of digital business operations.
+
+### Temporary assets and subscriptions {#temp-assets-subs}
+
+Temporary assets and subscriptions are notable applications of rental NFTs, offering accessibility to virtual experiences like high-end virtual casinos or curated digital fashion. These NFTs cater to diverse budgets, broadening audience reach. Subscription rentals extend to pools of digital assets, allowing users to pay monthly for a set number of items, fostering accessibility, user retention, and acquisition. Holders can rent out unused subscriptions, ensuring no loss for them, potential customer gains for the protocol, and a commitment-free trial for temporary holders. This showcases the adaptability and user-centric appeal of rental NFTs in diverse scenarios.
+
+## Smart contract design
+
+:::warning
+
+Transferring kiosks might result in unexpected behaviors while an asset is being rented. If you want to disallow kiosk transferring all together, consider using personal kiosks.
+
+:::
+
+The smart contract design leverages the Kiosk Apps standard to create a framework enabling transactions between lenders and borrowers. Both parties are required to install this extension to facilitate lending or borrowing activities. Moreover, the creator of the rented asset must establish a rental policy and a `ProtectedTP` object, granting authorization to the renting extension for managing assets subject to royalty enforcement.
+
+:::info
+
+This implementation is charging a rental fee based on days. You can re-purpose and update the logic to support charging per hour, or even seconds.
+
+:::
+
+## Move modules
+
+The NFT Rental example uses a single module, `nft_rental.move`. You can find the source for this file hosted in the [`sui-nft-rental` repository](https://github.com/MystenLabs/sui-nft-rental/blob/main/move/nft_rental/sources/nft_rental.move) on GitHub. The source code includes extensive comments to help you follow the example's logic and structure. 
```

Comment:
Is it possible to move this code into `~/sui/examples`?

Reply:
The reason this is important is testing -- if we make a change to the framework, or SDK or something, then CI will catch issues in examples and give us a chance to fix them.

Reply:
Sure but existing examples (at least [Asset Tokenization](https://github.com/MystenLabs/asset-tokenization) & [Satoshi](https://github.com/MystenLabs/satoshi-coin-flip)) are also not under `sui/examples`. Should those be moved as well?

Reply:
I feel like it could break things if we moved those (existing links to those repos)

Reply:
I see, yes this makes sense. We can definitely move rental code under `sui/examples` and delete the stand alone repo.
Here are some examples for the other 2 I mentioned:
- The [step by step full stack example created by SF](https://docs.sui.io/guides/developer/app-examples/coin-flip) as well as [SEs PoC app](https://satoshi-flip.mystenlabs.com/) link to the current [open sourced Satoshi Coin Flip repo](https://github.com/MystenLabs/satoshi-coin-flip).
- Also the [asset tokenization](https://docs.sui.io/guides/developer/advanced/asset-tokenization) docs point to [this repo](https://github.com/MystenLabs/asset-tokenization) currently.

Could potentially be missing other references (hopefully not though 😛)

Reply:
Note that we'll need to update the links here as well now that it's been moved into the `examples` directory.